draft-ietf-privacypass-auth-scheme.md

```diff
@@ -1027,4 +1047,30 @@ e="AAEADmlzc3Vlci5leGFtcGxlIIo-g6M9mABdLzC-9Bn6a_TNXGAF42sShbu0zNQPp
 LODAA5vcmlnaW4uZXhhbXBsZQ==", token-key="67H-0zgxA2HAjQx1dpaWcSluBem
 aF9eSbfwopT-r1In6wPgryoYkmmaPOlv6s3TJ",unknownChallengeAttribute="ig
 nore-me", max-age="10"
+
+token-type-0: 0x0000
+token-key-0: 856de3c710b892e7cca1ae5eb121af42ca8e779137a11224228c9b9
+9b0729bf84d5057d030000309b8f0d06ccffa17561f9eacd4c312e985a6bc60ffbea
+0610264dcb1726255313da81d665692686a1d8644f1516bf612cea009e6dff6d9a9a
+959fb538e1b5b2343c092992942382bdde22d5b324b1e4618ed21d7806286c2ce
+token-challenge-0: 0000acc3b25795c636fd9dd8b12982394abba8777d35978e8
+77fc8848892a217233045ac25a3d55c07c54efe6372973fee0073e77fc61bf19ab88
+0f20edf5d627502
+token-type-1: 0x0001
```

Comment:
Why would a client ignore this challenge?

Reply:
client would ignore Basic and 0x0000. They should not ignore 0x0001.

Reply:
Right, that's what I thought, but the text suggests that clients ought to ignore both.

Reply:
I seems github has not correctly assigned my comment https://github.com/ietf-wg-privacypass/base-drafts/pull/502#discussion_r1365766838. The text has been updated.